.github/pull_request_template.md

```diff
@@ -1,19 +1,25 @@
-## Done
+### Done
 
-[List of work items including drive-bys]
+- [Add clear and detailed changes description]
 
-## Issue / Card
+---
 
-Fixes #
-
-## QA
+### How to QA
 
-- Pull the branch
-- Run the site using the dotrun snap with `$ dotrun`
-- View the site locally in your web browser at: http://0.0.0.0:8004/
+- Visit the site in your browser at: [Replace with demo link]
```

Comment:
Let's save ourselves time and effort by just saying "Visit the demo in your browser, link commented below."